### DISCUSSION TITLE
Debugging R-Shiny

### DISCUSSION THREAD

**Discussion by dmeescor** [ManuelHentschel/VSCode-R-Debugger#135] [Q&A]
Are there any examples of how I can use this library to debug an R-Shiny application?

**Comment by wenzelkinsky** [comment]
This [stack overflow article](https://stackoverflow.com/questions/51542223/how-to-debug-r-shiny-code-with-vs-code) details using browser statements to do so, which tends to work fairly reliably albeit not ideal.
